src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -26,17 +26,30 @@
 		:open.sync="showSettings"
 		:show-navigation="true"
 		container="#content-vue">
+		<!-- description -->
 		<!-- Notifications settings -->
 		<AppSettingsSection
-			:title="t('spreed', 'Chat notifications')"
-			class="app-settings-section">
+			:title="t('spreed', 'Description')">
+			<Description
+				v-if="showDescription"
+				:editable="canFullModerate"
+				:description="description"
+				:editing="isEditingDescription"
+				:loading="isDescriptionLoading"
+				:placeholder="t('spreed', 'Add a description for this conversation')"
```

Comment:
since it's now in a dialog, having it say "Enter a description for this conversation" might be slightly more suitable

Reply:
done!